pyproject.toml

```diff
@@ -0,0 +1,95 @@
+[build-system]
+requires = ["scikit-build-core"]
+build-backend = "scikit_build_core.build"
+
+[tool.scikit-build]
+build.verbose = true
+logging.level = "INFO"
+
+[project]
+name = "catkit2"
+version = "0.0.1"
+authors = [
+  {name = "E.H. Por et al."},
+]
+description = "A library for controlling testbed hardware"
+requires-python = ">=3.7"
+dependencies = [
+]
+maintainers = [
+  {name = "Emiel Por", email = "epor@ucsc.edu"},
+  {name = "Remi Soummer", email = "soummer@stsci.edu"},
+  {name = "Iva Laginja", email = "iva.laginja@obspm.fr"}
+]
+readme = "README.md"
+license = {file = "LICENSE.rst"}
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Programming Language :: Python :: 3",
+]
+
+[project.urls]
+Homepage = "https://spacetelescope.github.com/catkit2"
+Documentation = "https://spacetelescope.github.io/catkit2"
+Repository = "https://spacetelescope.github.com/catkit2.git"
+
+[project.entry-points."catkit2.services"]
+aimtti_plp_device = "catkit2.services.aimtti_plp_device.aimtti_plp_device"
+aimtti_plp_device_sim = "catkit2.services.aimtti_plp_device_sim.aimtti_plp_device_sim"
+allied_vision_camera = "catkit2.services.allied_vision_camera.allied_vision_camera"
+bmc_deformable_mirror_hardware = "catkit2.services.bmc_deformable_mirror_hardware.bmc_deformable_mirror_hardware"
+bmc_deformable_mirror_sim = "catkit2.services.bmc_deformable_mirror_sim.bmc_deformable_mirror_sim"
+bmc_dm = "catkit2.services.bmc_dm.bmc_dm"
+bmc_dm_sim = "catkit2.services.bmc_dm_sim.bmc_dm_sim"
```

Comment:
Just fyi, I hope to wipe these with #285 today-ish.

Reply:
Fixed during rebase.